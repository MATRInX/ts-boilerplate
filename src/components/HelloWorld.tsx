import React, { SFC } from 'react';

export const HelloWorld: SFC<{}> = () => (
  <div>
    Hello World from TS!
  </div>
);

export default HelloWorld;