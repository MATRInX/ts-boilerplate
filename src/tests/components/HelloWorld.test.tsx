import React from 'react';
import { shallow } from 'enzyme';
import HelloWorld from '../../components/HelloWorld';

test('should correctly render Greeter', () => {
  const wrapper = shallow(<HelloWorld />);
  expect(wrapper).toMatchSnapshot();
});