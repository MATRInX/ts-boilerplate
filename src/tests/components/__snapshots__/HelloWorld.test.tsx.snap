// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Greeter 1`] = `
<div>
  Hello World from TS!
</div>
`;
